lib/net/validations.rb

```diff
@@ -9,6 +12,10 @@ module Validations
     class Error < RuntimeError
     end
 
+    class << self
+      include Net::Validations
```

Comment:
I'm not sure why this is needed but I'm not sure it's a good idea to make **all** instance methods available as module functions. Why not adding specific ones by `module_function` like existing `valid_mac?`?